README.md

```diff
@@ -17,3 +56,8 @@ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 
 [nightmare]: https://github.com/segmentio/nightmare
+[watchmen]: https://github.com/iloire/watchmen
+
+[add-service]: docs/add-new-service.png
+[select-ping-service]: docs/select-ping-service.png
+[script-file-path]: docs/script-file-path.png
```

Comment:
@joodies this is run under the expectation that it will be headless. So the STDOUT will return a success or error. `nightmare` actually runs a version of electron which is underpinned by chrome.